go.mod

```diff
@@ -108,7 +108,7 @@ require (
 	github.com/influxdata/httprouter v1.3.1-0.20191122104820-ee83e2772f69
 	github.com/influxdata/influx-cli/v2 v2.1.1-0.20210813175002-13799e7662c0
 	github.com/influxdata/influxdb-client-go/v2 v2.3.1-0.20210518120617-5d1fff431040 // indirect
-	github.com/influxdata/influxql v0.0.0-20180925231337-1cbfca8e56b6
+	github.com/influxdata/influxql v1.1.1-0.20210223160523-b6ab99450c93
```

Comment:
Are there any other meaningful changes this pulls in?

Reply:
Also do we need this change if we're not pulling in the WITH KEY work?

Reply:
Also we should probably have an issue to forward-port WITH KEY so we can keep track of what's done/not done/why.

Reply:
Bumping the InfluxQL version was necessary to be able to use the `PartitionExpr` from the InfluxQL package which was added recently. That's what really does the magic that fixes the tag values metaquery issue.

I created an issue for forwarding porting this and added a comment explaining what this PR partially ports: https://github.com/influxdata/influxdb/issues/22499

As far as InfluxQL changes, there's only a few - a couple of which seem somewhat meaningful:
- https://github.com/influxdata/influxql/pull/34 adds support for advanced bound parameters
- https://github.com/influxdata/influxql/pull/30 is a fix for https://github.com/influxdata/influxdb/issues/13929

Other than that, there's a [go version update](https://github.com/influxdata/influxql/pull/40), [optimization to use `strings.Builder`](https://github.com/influxdata/influxql/pull/39), and [a simple fix to code comments](https://github.com/influxdata/influxql/pull/44) - I wouldn't think these would have much user impact.

I suppose it would be good to have the `CHANGELOG` reflect these changes, although I'm not sure how that will work with the new changelog automation system.

Alternatively, I could just port the code out of InfluxQL in [this file](https://github.com/influxdata/influxql/blob/master/utils.go] into InfluxDB somewhere and use it that way. Do you have any strong feelings either way?

Reply:
I think we just take the influxql upgrade and call the version change out in the release notes.